cli/src/main.rs

```diff
@@ -56,12 +56,16 @@ fn main() -> Result<(), Box<dyn Error>> {
 
     let pretty_output = prettyplease::unparse(&syn::parse2(output).unwrap());
 
-    std::fs::write(&args.output_path, &pretty_output).unwrap_or_else(|_| {
-        panic!(
-            "Could not write the output to: {}",
-            args.output_path.display()
-        )
-    });
+    let output: &mut dyn Write = match &args.output_path {
+        Some(path) => &mut std::fs::File::create(path)
+            .unwrap_or_else(|_| panic!("Could not create the output file at: {}. Does its directory exist?", path.display())),
```

Comment:
This still uses a panic. I thought we'd want to just report the error and then gracefully exit?

Reply:
Do you want the exit code to be non-zero in that case?

Reply:
Other error cases panic in this binary. Do all of them need to be changed to reporting the error and gracefully exiting?

Reply:
Hmmm you're right. Ideally yes, but you don't have to do that. I'll fix that at some point later I think

Reply:
#69